api/config/config_local.json

```diff
@@ -105,5 +105,9 @@
     "host": "pmi-drc-api-test.appspot.com",
     "queueName": "rdrQueueTest",
     "exportObjectsPerTask" : 10
+  },
+  "captcha": {
+    "enableCaptcha": true,
+    "serverKeyFileName": "captcha-server-key.txt"
```

Comment:
Drive-by comment: I think I lost the last round of "battle" over this topic in a PR from a couple months back (@als364 and @jaycarlton were on that one), but I still don't believe that filenames like this really belong in our per-environment config. Our convention has always been to have consistent filenames across environments, with just the bucket name changing (see credentialsBucketName). Most of our key / credential filenames are code-constants in [CloudStorageServiceImpl](https://github.com/all-of-us/workbench/blob/master/api/src/main/java/org/pmiops/workbench/google/CloudStorageServiceImpl.java#L38)

I'm more than happy to be overruled if folks strongly disagree with that pattern. But it's important to note that we're actively introducing inconsistency in our system by storing key filenames in our config: some filenames will be stored in WorkbenchConfig, while most others continue to be code constants.

If we think there's benefit to migrating to the other approach, we should track a ticket & acknowledge the cost of migrating everything over there by filing a tech debt ticket.

Reply:
I agree we should migrate to the other approach, I don't think creating this new pattern should block this ticket, I do think creating said tech debt ticket should block this ticket.